TUTORIAL.md

```diff
@@ -0,0 +1,635 @@
+# Basic Student ORM
+
+This lab involves building a basic ORM for a Student object.
+
+## Objectives
+* Understand what an Object Relational Mapper(ORM) is
+* Gain ability to implement characteristics of an ORM when using a relational database management system (RDBMS) in a ruby program
+
+#### What is ORM?
+An ORM is an Object Relational Mapper. An ORM is basically a class that acts as an analogy for how instances of Objects in an object-oriented program correspond to rows in a database; that is it wraps the functionality of the database into our class.
```

Comment:
Another super convenient thing that ORMs do is wrap all of that SQL stuff and makes it so that Ruby writes our SQL content.